fido2/ctap.c

```diff
@@ -1597,45 +1685,37 @@ uint8_t ctap_cred_mgmt(CborEncoder * encoder, uint8_t * request, int length)
     }
     if (CM.cmd == CM_cmdRPBegin)
     {
-        curr_rk_ind = 0;
+        curr_rk_ind = -1;
+        curr_rp_ind = scan_for_next_rp(-1);
         rp_count = count_unique_rks();
```

Comment:
`count_unique_rks` is confusing as it is counting RPs, not RKs
can we reuse `scan_for_next_rp` and get rid of `count_unique_rks`?

Reply:
Good point, will refactor now